app/mod_blog/forms.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+
+from flask_wtf import Form
+from wtforms_components import DateRange
+from wtforms import StringField,BooleanField,PasswordField,SubmitField,TextAreaField,SelectField,DateField
+from wtforms.validators import DataRequired,Email,Length,EqualTo,optional
+from app.mod_user.models import User,Blog
+
+
+class BlogForm(Form):
```

Comment:
Use proper doc-string for , that will indicate the functionality of class or any module.